src/routes/dbInit/__mocks__/mockDOK.ts

```diff
@@ -0,0 +1,536 @@
+// tslint:disable
```

Comment:
I think everything in `src/routes/dbInit` should be moved into `src/routes/init` because there is no dbInit path and that is what the directories in `src/routes` specifies. Also you should rebase from dev because your PR does not have the `models` folder from the previous PR #8. You should be able to apply or edit those types to apply types to the mocks.